notebooks/data_selection.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    datacheck_files = glob.glob("/Users/moralejo/CTA/LST/RealData/datacheck/v0.9/DL1_datacheck_202*.h5")
</pre><p>This is a path on your machine. Better use the <code>/fefs/</code> part or something like:</p><p><code></code><code></p><p>dl1_check_path = Path(os.path.getenv("DATACHECK_PATH", "/fefs/.....") </p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
ok!